### PR TITLE
Feat: Avatar Component

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
An avatar component is added. Three different components are included 

1. Round Image Avatar 
2. Square Image Avatar
3. Text Round Avatar 

The code related to avatar is in div element with class of `demo-container` and `avatar.css` file

### CODE REVIEW

**Review by aryan-13 — APPROVED** [review]
Looks good to me!

File: components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+.avatar img,
+.avatar-sq img {
+  height: 100%;
+  object-fit: cover;
+  width: 100%;
+}
+
+.avatar,
+.avatar-text {
+  border-radius: 50%;
```
Comment (lannisterrr) [inline components/avatar.css:28]: this is being used in lot of components , so it can be a separate variable.

File: components/avatar.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+.avatar-md {
+  font-size: 2rem;
+  height: 8rem;
+  width: 8rem;
+}
+
+.avatar-sm {
+  font-size: 1rem;
```
Comment (lannisterrr) [inline components/avatar.css:57]: is it required to give font-size to figure element?

File: components/avatar.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  height: 5rem;
+  width: 5rem;
+}
+
+.avatar-xm {
```
Comment (prachi-sahani) [inline components/avatar.css]: did you mean .avatar-xs?

**Review by FarhanMobashir — COMMENTED** [review]
All good

File: components/variable.css [hunk 4]
```diff
@@ -1,4 +1,6 @@
 :root {
+  --bg-primary: #0b5ed7;
+
   --secondary-txt: #6c757d;
   --white-txt: white;
```
Comment (prachi-sahani) [inline components/variable.css]: here you can give hex value of white to maintain consistency

File: components/avatar.css [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+.avatar-sq img {
+  border-radius: 8px;
+}
+
+.avatar-text {
+  align-items: center;
+  background-color: var(--bg-primary);
+  display: flex;
+  justify-content: center;
+  text-align: center;
```
Comment (prachi-sahani) [inline components/avatar.css]: redundant? (since you are already centering the text using justify-content)

**Review by prachi-sahani — COMMENTED** [review]
Apart from the points mentioned, the code looks fine.

File: components/avatar.css (hunk 1 above)
Comment (junaidshaikh-js) [reply components/avatar.css:28]: Noted!

File: components/avatar.css (hunk 2 above)
Comment (junaidshaikh-js) [reply components/avatar.css:57]: It is for the text avatar not for image

File: components/avatar.css (hunk 3 above)
Comment (junaidshaikh-js) [reply components/avatar.css]: Oh! Didn't notice that. 

Thank you for pointing it out.

Comment (junaidshaikh-js) [reply components/avatar.css]: Fixed!

File: components/variable.css (hunk 4 above)
Comment (junaidshaikh-js) [reply components/variable.css]: Fixed!

File: components/avatar.css (hunk 5 above)
Comment (junaidshaikh-js) [reply components/avatar.css]: Fixed!

**Pull request merged**